modul/replay_ui.py

```
@@ -1,21 +1,36 @@
 from modul.replay_system import ReplayManager, ReplayPlayer
-from datetime import datetime
+try:
+    from modul.i18n import gettext
+except Exception:  # pragma: no cover - fallback when i18n unavailable
+    def gettext(k):
+        return k
+
+try:
+    from modul import input_utils
+except Exception:  # pragma: no cover - minimal stub for tests
+    class _InputUtilsStub:
```

Pylint (4.0.4)

[refactor] Too few public methods (1/2)

(R0903)